src/components/fields/file-field/FileFieldGroup.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import classNames from 'classnames';
+import {RenderGroup} from '../../RenderGroup';
+import {Message} from '../messages/Message';
+import {FileSelect} from './FileSelect';
+import {fieldGroupPropTypes} from '../propTypes';
+
+/**
+ * Encapsulates a complete File field group
+ */
+export class FileFieldGroup extends React.PureComponent {
+
+	/**
+	 * Create a FileFieldGroup component
+	 *
+	 * @param {Object} props
+	 */
+	constructor(props) {
+		super(props);
+		this.onChange = this.onChange.bind(this);
+		this.onSelect = this.onSelect.bind(this);
+	}
+
+	/**
+	 * Handle direct change events
+	 * @param {String|number} newValue
+	 */
+	onChange(newValue) {
+		this.props.onValueChange(newValue);
+	}
+
+	/**
+	 * Handle when the option is chosen
+	 * @param {String|number} value
+	 */
+	onSelect(value) {
+		this.props.onValueChange(value);
+	}
+
+	/**
+	 * Render FileFieldGroup component
+	 * @return {*}
+	 */
+	render() {
+		return (
+			<div
+				className={classNames(
+					FileFieldGroup.classNames.fieldWrapper,
+					RenderGroup.classNames.fieldWrapper,
+					this.props.className
+				)
+				}
+			>
+			{this.props.message.message &&
+				<Message
+					message={this.props.message}
+				/>
+			}
+				<label
```

Comment:
Please use the FieldGroup.Label component

https://github.com/CalderaWP/caldera-components/blob/3684e109b6127a6d561aff9baddf3107f0f3a03c/src/components/fields/FieldGroup.js#L126

https://github.com/CalderaWP/caldera-components/blob/3684e109b6127a6d561aff9baddf3107f0f3a03c/src/components/fields/magic-select/MagicFieldGroup.js#L164-L167